subsys/bluetooth/services/nus.c

```diff
@@ -16,16 +16,14 @@ LOG_MODULE_REGISTER(bt_gatt_nus, CONFIG_BT_GATT_NUS_LOG_LEVEL);
 static struct bt_gatt_nus_cb nus_cb;
 
 static bool is_notification_enabled(struct bt_conn *conn,
-				    const struct bt_gatt_ccc_cfg *ccd)
+				    const struct bt_gatt_attr *attr)
 {
-	const bt_addr_le_t *conn_addr = bt_conn_get_dst(conn);
+	const bt_addr_le_t *peer = bt_conn_get_dst(conn);
+	const struct _bt_gatt_ccc *ccc = attr->user_data;
```

Comment:
Since this is considere "internal" I will move this functionality upstream. And use a stack function instead.